src/app/pre-common/omsorgstilbud-periode/OmsorgstilbudPeriodeForm.tsx

```diff
@@ -0,0 +1,142 @@
+                                <div style={{ maxWidth: '20rem' }}>
+                                    <FormBlock>
+                                        <FormComponents.DateIntervalPicker
+                                            fromDatepickerProps={{
+                                                label: intlHelper(intl, 'omsorgstilbudPeriodeForm.fraOgMed.label'),
+                                                name: FormFields.fom,
+                                                disableWeekend: true,
+                                                fullScreenOnMobile: true,
+                                                dayPickerProps: {
+                                                    initialMonth: rammePeriode.from,
+                                                },
+                                                minDate: rammePeriode.from,
+                                                maxDate: to || rammePeriode.to,
+                                                validate: getDateRangeValidator({ required: true, onlyWeekdays: true })
+                                                    .validateFromDate,
+                                            }}
+                                            toDatepickerProps={{
+                                                label: intlHelper(intl, 'omsorgstilbudPeriodeForm.tilOgMed.label'),
+                                                name: FormFields.tom,
+                                                disableWeekend: true,
+                                                fullScreenOnMobile: true,
+                                                minDate: from || rammePeriode.from,
+                                                maxDate: rammePeriode.to,
+                                                dayPickerProps: {
+                                                    initialMonth: from || rammePeriode.from,
+                                                },
+                                                validate: getDateRangeValidator({ required: true }).validateToDate,
+                                            }}
+                                        />
+                                    </FormBlock>
+                                </div>
+
+                                <FormBlock>
+                                    <FormComponents.InputGroup
+                                        legend={intlHelper(intl, 'omsorgstilbudPeriodeForm.tidFasteDager.label')}
+                                        // validate={() => validateFasteArbeidstimerIUke(tidFasteDager, intlValues)}
+                                        validate={() => validateOmsorgstilbudIUke(tidFasteDager)}
+                                        name={'fasteDager_gruppe' as any}>
+                                        <TidUkedagerInput name={FormFields.tidFasteDager} />
```

Comment:
Validator mangler her (ser det også mangler på arbeidstid). Så vi bør prøve å få inn den. Nå er det tillatt med ugyldige tegn. For å få den inn på arbeidstid legges denne inn
`validator={getArbeidstimerFastDagValidator}`. Tilsvarende må lages for omsorgstilbud.